docs/content/uijson_format/json_objects.rst

```diff
@@ -0,0 +1,299 @@
+
+.. code-block:: json
+
+   {
+   "meshType": ["{202C5DB1-A56D-4004-9CAD-BAAFD8899406}" ,
+      "{6A057FDC-B355-11E3-95BE-FD84A7FFCB88}"],
+   "main": true,
+   "label": "Select Points or Curve",
+   "value": ""
+   }
+
+.. figure:: ./images/object_param.png
+
+
+.. _data_parameter:
+
+Data parameter
+==============
+
+Creating a parameter to choose an object's data requires extra members:
+
+dataType ``str``
+   Describes the type of data to filter. One or more of these key words: ``Integer``, ``Float``, ``Text``, ``Referenced``, ``Vector``, ``DataTime``, ``Geometric``, ``Boolean``, or ``Text``
+association ``str``
+   Describes the geometry of the data. One or more of these key words: ``Vertex``, ``Cell``, or ``Face``
+parent ``str``
+   Either a *UUID* of the parent or the name of the :ref:`Object parameter <object_parameter>` json object to allow the user to choose the mesh.
+isValue ``bool``
+   Describes whether to read the **value** (``float``) or **property** (``str``) member. If not given, the value member is an *UUID* and is considered a *drop-down data parameter*. If this member is given along with **property**, then an icon changing from float to drop-down list will appear. Geoscience ANALYST Pro will update this value depending on the style choice (``float`` or ``str``)
+property ``str``
+   Data *UUID*  that is selected when **isValue** is present.  Geoscience ANALYST Pro will update this value on export.
+min ``float``
+    Optional minimum value allowed for the value. The default is the minimum numeric limits of float.
+max ``float``
+    Optional maximum value allowed for the value. The default is the maximum numeric limits of float.
+precision ``int``
+    Optional number of decimal places for the value.
+
+
+Drop-down data parameter
+------------------------
+In this example, the object parameter *data_mesh* is also given for reference.
+
+.. code-block:: json
+
+   {
+   "data_channel": {
+   "main": true,
+   "association": "Vertex",
+   "dataType": "Float",
+   "label": "Data channel",
+   "parent": "data_mesh",
+   "value": ""
+   },
+   "data_mesh": {
+   "main": true,
+    "meshType": ["{202C5DB1-A56D-4004-9CAD-BAAFD8899406}" ,
+      "{6A057FDC-B355-11E3-95BE-FD84A7FFCB88}"],
+   "main": true,
+   "label": "Select Points or Curve",
+   "value": ""
+   }
+   }
+
+
+.. figure:: ./images/data_param.png
+
+
+
+Data or value parameter
+-----------------------
+In some cases, a simple float will do. The use of the member **isValue** and **property** together allows the choice within the UI. In this example, the **property** member will turned on to begin with. The **value** will default to 1.0 and show up when the icon is clicked. The **uncertainty channel** object depends on the **data_mesh** object. The combobox will filter data from the chosen object that is located on the vertices and is float (pi icon).
```

Comment:
```suggestion
In some cases, a parameter may take it's data from an object, or a simple float will do. The use of the **isValue** and **property** fields allows the UI to switch between these to cases. In this example, the **property** field will be active (**isValue**: false) to begin with, allowing the choice of a set of data defined on the vertices of the parent object (**data_mesh**). When the pi icon is clicked the dropdown box will be replaced with a float box defaulted to the **value** 2 and the **isValue** field will be made true.
```

Reply:
We should show a second image (before and after)

Reply:
Yes, that is better I think.